src/frontends/tensorflow_common/src/op/binary_op.cpp

```diff
@@ -142,6 +142,30 @@ OutputVector translate_mul_op(const NodeContext& node) {
     set_node_name(node.get_name(), result);
     return {result};
 }
+
+OutputVector translate_sub_op(const NodeContext& node) {
+    default_op_checks(node, 2, {"Sub"}, true);
+    auto lhs = node.get_input(0);
+    auto rhs = node.get_input(1);
+
+    auto result = make_shared<v1::Subtract>(lhs, rhs);
```

Comment:
Again, this is incorrect for complex type case because you pass `ComplexTypeMark` outputs to `Subtract`.

Reply:
Please check how we handle it for `Mul` operation: https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/tensorflow_common/src/op/binary_op.cpp#L95